src/controller/python/chip-device-ctrl.py

```diff
@@ -816,6 +816,51 @@ def zcl_command(
     except Exception as e:
         return __get_response_dict(status = StatusCodeEnum.FAILED, error = str(e))
 
+def zcl_read_attribute(
+        cluster: str,
+        attribute: str,
+        node_id: int,
+        endpoint_id: Optional[int] = 0,
+        group_id: Optional[int] = 0):
+    """Generic RPC call to read ZCL cluster attributes
+    zclread command format: zclread <Cluster> <Attribute> <NodeId> <EndpointId> <GroupId> [arguments]
+    
+    Args:
+        cluster (str): Name of cluster
+        command (str): Command to be run for the mentioned cluster
+        node_id (int): node_id assigned to the DUT
+        endpoint_id (int): Optional, endpoint id
+        group_ip (int): Optional, group id
```

Comment:
No need to include types in the Args list since you include type annotations in the function call. Please add punctuation.

```
Args:
        cluster: Name of cluster.
        command: Command to be run for the mentioned cluster.
        node_id: node_id assigned to the DUT.
        endpoint_id: endpoint id.
        group_ip: group id.
```

Also, can you be more descriptive for the group id and endpoint id? Asking because I am not sure of what they are. If you cannot think of a way to describe them just leave it.

Reply:
I am not sure about the exact description for endpoint and groupid will check and update

Reply:
Please remove the `Optional,` from their descriptions. Already clear in the type annotations.